### PR TITLE
feat: Looker 21.14 release

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

File: python/looker_sdk/rtl/api_methods.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
         Sequence[model.Model],
     ]
 ]
-TStructure = Optional[Union[Type[str], serialize.TStructure]]
+TStructure = Optional[Union[Any, Type[str], serialize.TStructure]]
```
Comment (joeldodge79) [inline python/looker_sdk/rtl/api_methods.py:47]: the new `api_spec` method returns `Any` so needed to add that to the union type here

Comment (jkaster) [reply python/looker_sdk/rtl/api_methods.py:47]: Yeah, it's now (correctly) a JSON document

File: release-please-config.json [hunk 2]
```diff
@@ -16,7 +17,7 @@
     "packages/sdk-codegen-scripts": { "release-as": "" },
     "packages/sdk-codegen-utils": { "release-as": ""},
     "packages/sdk-node": { },
-    "packages/sdk-rtl": { "release-as": "" },
+    "packages/sdk-rtl": { "release-as": "21.0.20" },
```
Comment (jkaster) [inline release-please-config.json:20]: I'm sure this was a manual fix. Maybe we should rethink this behavior for the rtl package.

**Review by jkaster — APPROVED** [review]
LGTM

File: release-please-config.json (hunk 2 above)
Comment (joeldodge79) [reply release-please-config.json:20]: sorry, forgot to annotate this. tldr I'm back to desiring to keep these as `21.0.x` till we re-name and then just let it semver? see https://github.com/looker-open-source/sdk-codegen/pull/802#discussion_r699397245

**Pull request merged**